hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -117,7 +118,13 @@ private void initTableNameVars(HiveSyncConfig config) {
           this.roTableName = Option.empty();
           break;
         case MERGE_ON_READ:
-          this.snapshotTableName = tableName + SUFFIX_SNAPSHOT_TABLE;
+          if (config.getBoolean(HIVE_SKIP_RT_SUFFIX_FOR_READ_SNAPSHOT_TABLE)
+                  && config.getBoolean(HIVE_SKIP_RO_SUFFIX_FOR_READ_OPTIMIZED_TABLE)) {
+            throw new HoodieHiveSyncException("can not be set both skip _rt and _ro are true at the same time!");
+          }
```

Comment:
> can not be set both skip _rt and _ro are true 

Can not skip _ro and _rt suffix both at the same time

Reply:
ok，thanks！